README.md

```diff
@@ -21,7 +21,7 @@ There are a few different ways to install git (from source or for Linux) but the
 ```sh
 $ git config --global user.name "User Name"
 
-$ git config --global user.email "email"
+$ git config --global user.email "email address"
```

Comment:
hi,
Can you add "email address INFO", instead of "email address"?

Thanks!

Reply:
ok ill do it by today